corehq/apps/accounting/tests/test_customer_invoicing.py

```diff
@@ -120,80 +132,62 @@ def tearDown(self):
 
     @classmethod
     def tearDownClass(cls):
-        cls.domain.delete()
-        cls.domain2.delete()
-        cls.domain3.delete()
-        cls.domain_community.delete()
+        cls.main_domain.delete()
+        cls.non_main_domain1.delete()
+        cls.non_main_domain2.delete()
+        cls.non_main_domain3.delete()
 
         super(BaseCustomerInvoiceCase, cls).tearDownClass()
 
 
 class TestCustomerInvoice(BaseCustomerInvoiceCase):
 
     def test_multiple_subscription_invoice(self):
-        invoice_date = utils.months_from_date(self.subscription.date_start,
-                                              random.randint(3, self.subscription_length))
-        calculate_users_in_all_domains(invoice_date)
-        tasks.generate_invoices_based_on_date(invoice_date)
-
-        self.assertEqual(CustomerInvoice.objects.count(), 1)
-        invoice = CustomerInvoice.objects.first()
-        self.assertGreater(invoice.balance, Decimal('0.0000'))
-        self.assertEqual(invoice.account, self.account)
-
-        num_product_line_items = invoice.lineitem_set.get_products().count()
-        self.assertEqual(num_product_line_items, 2)
-
-        num_feature_line_items = invoice.lineitem_set.get_features().count()
-        self.assertEqual(num_feature_line_items, self.subscription.plan_version.feature_rates.count() +
-                         self.sub2.plan_version.feature_rates.count())
-
-    def test_only_invoice_active_subscriptions(self):
```

Comment:
Why was this test removed?

Reply:
This test assumes, under one customer billing account, there are two domains, one has active subscription, another's subscription has already ended. But in reality, if a subscriptions simply ends, without being upgrade to another subscription, Ops will remove that domain from this customer billing account. Thus I feel this test is a bit redundant.